index.html

```diff
@@ -1077,6 +1173,32 @@ <h2><dfn>CaptureStartFocusBehavior</dfn>
         </div>
       </section>
 
+      <section>
+        <h2><dfn>DisplaySurfaceChangeCallback</dfn></h2>
+        <p>
+          Used to receive a new {{MediaStream}} if the [=display
+          surface=] associated with a [=capture-sessions=] changes and
+          a callback has been set on the associated
+          {{CaptureController/setDisplaySurfaceChangeCallback()}}
```

Comment:
nit: s/capture-sessions/capture-session

But lease consider the alternative of "Invoked whenever the MediaStream associated with a given capture-session changes." That it's not invoked if not set seems to me a bit out of scope for the current section.

Reply:
Fixed nit and removed the part about a callback having be set.

I don't think we should talk about a MediaStream being associated with a capture-session since the MediaStream might be cloned/stopped etc. Instead we should talk about the capture-source-set. I have updated the wording a bit.